kratos/geometries/brep_curve_on_surface.h

```diff
@@ -338,6 +338,70 @@ class BrepCurveOnSurface
             mCurveNurbsInterval.GetT0(), mCurveNurbsInterval.GetT1());
     }
 
+    ///@}
+    ///@name IsInside
+    ///@{
+
+    /// returns if rPointLocalCoordinates[0] is inside -> 1 or outside -> 0
+    int IsInsideLocalSpace(
+        const CoordinatesArrayType& rPointLocalCoordinates,
+        const double Tolerance = std::numeric_limits<double>::epsilon()
+    ) const override
+    {
+        const double min_parameter = mCurveNurbsInterval.MinParameter();
+        if (rPointLocalCoordinates[0] < min_parameter) {
+            return 0;
+        }
```

Comment:
Should be `} else if (...) {`

Idem for the other ones

Reply:
If you don't insist I would personally prefer to keep it as it is. Otherwise, it'll be very dense to read...

Reply:
It is the style guide...

Reply:
I think there might be a possible variation? ;)

Reply:
My heart is bleeding, but I did it ;)